packages/backend/src/models/ProjectModel/ProjectModel.ts

```diff
@@ -383,4 +388,41 @@ export class ProjectModel {
             })
             .where('project_uuid', projectUuid);
     }
+
+    async getCacheExplores(
+        projectUuid: string,
+    ): Promise<(Explore | ExploreError)[]> {
+        const explores = await this.database(CacheExploresTableName)
+            .select(['explores'])
+            .where('project_uuid', projectUuid)
+            .limit(1);
+        if (explores.length > 0) return explores[0].explores;
+        return [];
+
+        /* 
+        TODO Should we return error if nor esults ? 
```

Comment:
what do you think ?

Reply:
Not sure what's better! 

I think returning `undefined | null` or raising an error is most useful to indicate whether there was a cache-miss vs. a hit but it contained an empty list